src/components/breadcrumb/breadcrumb.js

```diff
@@ -12,7 +12,15 @@ const Breadcrumbs = (props) => {
     if (props.categories.length > 0) {
       return props.categories.map((cat, i) => {
         return (
-          <Link key={i} to={cat.link} className="flex-1">
+          <Link
+            key={i}
+            to={cat.link}
+            style={{
+              'text-decoration': 'underline',
+              'text-decoration-style': 'dotted',
+            }}
```

Comment:
Try this

https://tailwindcss.com/docs/text-decoration-style#setting-the-text-decoration-style